tests/www/views/test_views.py

```diff
@@ -62,6 +62,13 @@ def test_configuration_expose_config(admin_client):
     check_content_in_response(["Airflow Configuration"], resp)
 
 
+@mock.patch.dict(os.environ, {"AIRFLOW__WEBSERVER_CONFIG_FILE": "{AIRFLOW_HOME}/config/webserver_config.py"})
```

Comment:
@jedcunningham I was trying to get a thorough UT here where I override the default location of the webserver config file. However, the mock.patch here doesn't seem to get honored. Need some help here.

Reply:
Trying something like this too 
```
@mock.patch("airflow.configuration.conf")
def test_webserver_configuration_config_file(conf):
    conf.get.return_value = "/tmp/webserver_config.py"

    assert WEBSERVER_CONFIG == "/tmp/webserver_config.py"
```

I think I am missing something trivial here. Any idea?

Reply:
You're missing an `_`, try with `AIRFLOW__WEBSERVER__CONFIG_FILE`.